pkg/network/namescheme/networknamescheme_suite_test.go

```diff
@@ -13,22 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * Copyright 2021 Red Hat, Inc.
+ * Copyright 2022 Red Hat, Inc.
```

Comment:
All changes to this file look very weird to me.

Reply:
I think I switched between `networknamescheme_test.go` and `networknamescheme_suite_test.go`, oops! will fix

Reply:
Done